src/nvim/os/env.c

```diff
@@ -317,10 +325,17 @@ void init_homedir(void)
       }
     }
   }
+
   if (var == NULL) {
     var = os_getenv("USERPROFILE");
```

Comment:
`uv_os_homedir()` first checks `USERPROFILE` at [line 1133 of src/win/util.c](https://github.com/libuv/libuv/blob/747a0f34d7ed62f78103a444ef657db866dff3fd/src/win/util.c#L1133). Therefore, this can be deleted.

Reply:
Done